main.go

```diff
@@ -547,7 +547,7 @@ func check_handler(w http.ResponseWriter, r *http.Request) {
 	checkTemplate.Execute(w, c)
 }
 
-func index_handler(w http.ResponseWriter) {
+func index_handler(w http.ResponseWriter, _ *http.Request) {
```

Comment:
Here is the fix which solves the compilation error

Reply:
@pattmax00 RE: which go version I was using: 

I might have been using `go version go1.19.4 linux/amd64` on Void Linux. I'm not sure if this is accurate as I might have upgraded the go binary since then and didn't check at the time.